js/src/features/human-resouce/components/Absence/Abcence.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback } from "react";
+import { List, Tag, Tooltip } from "antd";
+import {
+  CheckCircleOutlined,
+  CloseCircleOutlined,
+  EditOutlined,
+} from "@ant-design/icons";
+import { Absence as AbsenceModel } from "HumanResourceModels";
+
+interface Props {
+  absence: AbsenceModel;
+  canEdit: boolean;
+  approve: (id: string) => any;
```

Comment:
The result value should be not 'any' but 'void'

Reply:
And below too

Reply:
fixed